src/DocumentFormat.OpenXml.Features/ParagraphId/ParagraphIdOptions.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace DocumentFormat.OpenXml.Features
+{
+    /// <summary>
+    /// Options to declare how paragraph ids should be generated.
+    /// </summary>
+    public class ParagraphIdOptions
+    {
+        /// <summary>
+        /// Gets a value indicating whether or not document changes should be tracked.
+        /// </summary>
+        public bool TrackDocumentChanges { get; init; } = true;
+
+        /// <summary>
+        /// Gets a value indicating whether or not paragraph ids should be autogenerated on elements.
+        /// </summary>
+        public bool AutoGenerateIds { get; init; } = true;
+
+        /// <summary>
+        /// Gets a value indicating whether or not uniqueness should be guaranteed on elements added to the document.
+        /// </summary>
+        public bool EnsureUniqueness { get; init; }
```

Comment:
Switched it to true as that is a better default. It could be that the first one wins method of setting para ids could break something. Ideally, we'd have a way to replace that strategy but for simplicity for now, this will just disable it. I could envision an additional callback or something when duplicates are found.